tests/signature_tests/test_ipsignature.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from scapy.all import rdpcap
+from scapy.layers.inet import IP
+from src.signature_detection.ip_signature import IPSignature
+from pathlib import Path
+
+
+class TestIPSignature(unittest.TestCase):
+    def setUp(self) -> None:
+        self.ip_signature = IPSignature("192.168.1.0/24")
+        packets = rdpcap(str(Path("../tests/test_data/test_tcpdump2.pcap").resolve()))
+        self.src_malicious_ip_packet = packets[0]
+        self.dst_malicious_ip_packet = packets[1]
+        self.src_benign_ip_packet = packets[2]
+        self.dst_benign_ip_packet = packets[3]
+        self.src_malicious_ip_packet[IP].src = "62.210.205.141"
```

Comment:
What are these IPs?

Reply:
Known malicious IPs that hit on virustotal